### ISSUE TITLE
Fantastic work! It would be appreciated that you upload the training code dealing with Hanzi or Imagenet.

### ISSUE THREAD

**Issue by DLwbm123** [czbiohub/noise2self#5]

**Comment by batson** [comment]
Hello! The training code for those datasets is somewhat tuned for our compute environment, and is primarily data loading and training monitoring boilerplate. The essential step--using `masker.mask`--is demonstrated in the example Jupyter notebooks.

If you'd like the training data, however, the Hanzi dataset is available on [github](github.com/batson/hanzi).
